src/components/loader/index.tsx

```diff
@@ -0,0 +1,15 @@
+import * as React from "react";
+import styles from "../../css-modules/loader.module.css";
+
+function Loader() {
+  return (
+    <>
+      <div className={styles.spinner} />
+      <br />
```

Comment:
Usually using br elems is not the best practice

Reply:
resolved